src/pages/index.js

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import Header from '../components/header/Header';
+import UserDetail from '../components/userdetail/UserDetail';
+import PersonalInfo from '../components/preview/PersonalInfo';
 
 const App = () => {
-  return <h1>Resume Builder</h1>;
+  // App state
+  const [preview, setPreview] = useState(false);
+
+  // Data source
+  const personalInfo = {
+    name: { data: 'Ribby McFroggy', visibility: true },
+    summary: { data: 'Hlelo', visibility: true },
```

Comment:
I can understand it is a dummy data but you have got a typo here

Reply:
[form-preview bd6005b] form-preview:#2 - fixed typo